docs/source/index.rst

```diff
@@ -0,0 +1,291 @@
+cluster_generator
+=================
+
+|yt-project| |docs| |testing| |Github Page| |Pylint| |coverage| |ncodes|
+
+.. raw:: html
+
+   <hr style="height:10px;background-color:black">
+
+The `Cluster Generator Project <https://eliza-diggins.github.io/cluster_generator>`_ (CGP) is a cross-platform Python library for generating initial conditions of galaxy clusters for N-body / hydrodynamics codes.
```

Comment:
I'm ambivalent about this name. I think it should be "CG" if we're going to abbreviate it.

Reply:
I've swapped CGP -> CG in the most recent commit. Also corrected links to documentation hosted under my account. I figure I'd rather have broken links for a while than miss a link that jumps between two very different versions of the documentation!

I do believe that the version of the documentation currently hosted at https://eliza-diggins.github.io/cluster_generator is only 1 pull request ahead of this one if you want to poke around!